megamek/src/megamek/client/ui/panels/phaseDisplay/ReportDisplay.java

```diff
@@ -271,4 +287,33 @@ public void removeAllListeners() {
         clientgui.getClient().getGame().removeGameListener(this);
         clientgui.boardViews().forEach(bv -> bv.removeBoardViewListener(this));
     }
+
+    /**
+     * Shows the Nova CEWS network management dialog.
+     */
+    private void showNovaNetworkDialog() {
+        LOGGER.debug("Opening Nova CEWS Network Management Dialog");
```

Comment:
These log messages are probably unnecessary (or at least, we don't typically do this to my knowledge)

Reply:
Addressed as part of https://github.com/MegaMek/megamek/pull/7635/commits/3ff983a99dc7e832f0d045cbe5ddc06d9aa89f3c